testnet/launcher/eth2network/docker.go

```diff
@@ -50,7 +50,7 @@ func (n *Eth2Network) Start() error {
 }
 
 func (n *Eth2Network) IsReady() error {
-	timeout := 10 * time.Minute
+	timeout := 20 * time.Minute
```

Comment:
Why do we need to increase the timeout here?
Was there a performance degradation?

Reply:
I had slow internet, and it took ages to download the eth2 binaries